src/static/js/components/viewer.js

```diff
@@ -0,0 +1,37 @@
+export default class Viewer {
+  constructor(elem, params = {}) {
+    this.elem = elem;
+    this.params = params;
+
+    this.cssElemType = this.params.cssElemType || '';
+    this.cssElemImage = this.params.cssElemImage || '';
+
+    this.construct();
+  }
+
+  construct() {
+    this.elType = this.elem.getElementsByClassName('JS-View-Type')[0];
+    this.elImage = this.elem.getElementsByClassName('JS-View-Image')[0];
+
+    this.init();
+    this.ready();
+  }
+
+  init() {
+    var _this = this;
```

Comment:
add console.log(this.elType) here and look what is going on.

Reply:
https://nimb.ws/PB4bxX